web/src/main/java/com/epam/rft/atsy/web/controllers/CandidateCreationController.java

```diff
@@ -30,11 +31,16 @@
    * @param candidateId the identifier of the candidate whose data should be given back
    * @return a ModelAndView object which contains the all candidate information and the name of the
    * page that handles the candidate data
+   * @throws com.epam.rft.atsy.web.exceptionhandling.CandidateNotFoundException
```

Comment:
You could use CandidateNotFoundException in @throws instead of the fully qualified name (com.epam.rft.atsy.web.exceptionhandling.CandidateNotFoundException).

Also please specify in the JavaDoc in which case this exception occurs.